.github/workflows/integration-test.yml

```diff
@@ -2,19 +2,24 @@ name: Integration Tests
 
 on: [push, pull_request]
 
+env:
+  TEST_BROWSER_HEADLESS: 1
+  CI: 1
+
 jobs:
   tests:
     name: Run integration tests
     runs-on: ubuntu-latest
     steps:
       - name: Download OpenSearch Core
         run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.6/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-1.3.6-linux-x64.tar.gz
```

Comment:
I believe these 2.4.0 version updates shouldn't be part of this PR

Reply:
Yeah, I am just trying to get anything to work at this point. So I just copied the entire file to make sure that it was not a missing line somewhere that was causing the issue. I will be going back through to pair down the changes to those necessary once I get anything working.